### PR TITLE
AC: Updated classification_f1-score metric for MRPC task

### PR DESCRIPTION
**Pull request by Anna-Mironova**
- Fixed filling confusion matrix
- Added pos_label argument to specify label to report

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/metrics/README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Supported representations: `ClassificationAnnotation`, `TextClassificationAnnota
 * `label_level_recognition_accuracy` - [label level recognition accuracy](https://dl.acm.org/doi/abs/10.1145/1143844.1143891) metric for text line character recognition task using [editdistance](https://pypi.org/project/editdistance/). Metric is calculated as a percentage. Direction of metric's growth is higher-better. Supported representations: `CharacterRecognitionAnnotation`, `CharacterRecognitionPrediction`.
 * `classification_f1-score` - [F1 score](https://en.wikipedia.org/wiki/F1_score) metric for classification task. Metric is calculated as a percentage. Direction of metric's growth is higher-better. Supported representations: `ClassificationAnnotation`, `TextClassificationAnnotation`, `ClassificationPrediction`.
   * `label_map` - the field in annotation metadata, which contains dataset label map (Optional, should be provided if different from default).
+  * `pos_label` - class to report during metric calculation (Optional).
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/README.md:31]: Please describe what reported if this option is not provided

File: tools/accuracy_checker/accuracy_checker/metrics/classification.py [hunk 2]
```diff
@@ -288,6 +293,10 @@ def evaluate(self, annotations, predictions):
         )
         if self.profiler:
             self.profiler.finish()
+
+        if self.pos_label is not None:
+            return f1_score[self.pos_label]
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/classification.py:298]: metric meta names should be modified in appropriate way (contains only this class)

Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/metrics/classification.py:299]: Done

File: tools/accuracy_checker/accuracy_checker/metrics/README.md (hunk 1 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/metrics/README.md]: Done

**Review by eaidova — APPROVED** [review]

**Pull request merged**